Close-the-element-using-the-button/test.js

```diff
@@ -0,0 +1,12 @@
+const divCollectionPane = document.querySelectorAll(".pane");
+
+divCollectionPane.forEach((item) => {
+  const button = document.createElement("button");
+  button.classList.add("remove-button");
+  button.textContent = "[x]";
+  item.append(button);
+
+  button.addEventListener("click", function () {
+    item.style.display = "none";
```

Comment:
If you want to remove pane from DOM then use 
```
item.remove()
```
https://developer.mozilla.org/en-US/docs/Web/API/Element/remove

When you use `display: none`, the element is saved in the DOM tree